openmc/weight_windows.py

```diff
@@ -18,7 +20,7 @@ class WeightWindows(IDManagerMixin):
     """Mesh-based weight windows
 
     This class enables you to specify weight window parameters that are used in
-    a simulation. Multiple sets of weight windows can be defined for different
+    a simulation. Multiple sets of weight windows can be defined for differen  t
```

Comment:
```suggestion
    a simulation. Multiple sets of weight windows can be defined for different
```
looks like a some spaces got added here